src/cirq_iqm/iqm_sampler.py

```diff
@@ -94,7 +94,8 @@ def run_sweep(
         mapped = program
         if self._qubit_mapping is not None:
             # apply the qubit_mapping
-            qubit_map = {cirq.NamedQubit(k): cirq.NamedQubit(v) for k, v in self._qubit_mapping.items()}
+            qubit_map: Union[dict['cirq.Qid', 'cirq.Qid'], Callable[['cirq.Qid'], 'cirq.Qid']] = \
```

Comment:
Does mypy recognize just `dict['cirq.Qid', 'cirq.Qid']` matching the `Union` function arguments?

Reply:
Good call, it does. Updated the file, thanks!